NOpenType/OpenTypeReader.cs

```diff
@@ -2,16 +2,22 @@
 using NRasterizer.Tables;
 using System;
 using System.Collections.Generic;
-using System.IO;
-using System.Linq;
+using System.IO; 
 
 namespace NRasterizer
 {
     public class OpenTypeReader
     {
         private TableEntry FindTable(IEnumerable<TableEntry> tables, string tableName)
         {
-            return tables.Single(t => t.Tag == tableName);
+            foreach (TableEntry t in tables)
+            {
+                if(t.Tag == tableName)
+                {
+                    return t;
+                }
+            }
+            return null;
```

Comment:
Single raises an exception if there are no matchin entries. Instead make a function that searcher in an enumerable.